opendbc/car/volkswagen/mqbcan.py

```diff
@@ -108,9 +108,9 @@ def create_acc_accel_control(packer, bus, acc_type, acc_enabled, accel, acc_cont
   if starting:
     acc_hold_type = 4  # hold release / startup
   elif esp_hold:
-    acc_hold_type = 3  # hold standby
+    acc_hold_type = 1  # hold request (at stop)
```

Comment:
these were flipped from how stock ACC uses them. the car will still respect either value and stop, but sometimes the car is late to release ESP_confirmation (ESP_Haltebestaetigung) when transitioning directly from 3 -> 4.

on acc type 2 this is fine, but on acc type 1 this can easily cause faults if the hold confirmation is retained for too long.